### DISCUSSION TITLE
Choosing Application Name

### DISCUSSION THREAD

**Discussion by mvchn** [trngl8/cow#16] [Ideas]
Simple name will be better

**Comment by mvchn** [comment]
Іншими словами, ми не знаємо як аплікація буде називатись. Ми ж будуємо не корову по суті, а ферму.

**Comment by mvchn** [comment]
Я думаю, це може називатись "The Farming Game Framework"

**Reply by Bujhm** [reply]
Может без слова Game?

**Reply by Bujhm** [reply]
Just "The Farm" )
